bin/acore-docker-build

```diff
@@ -2,8 +2,8 @@
 
 docker build -t acbuild -f docker/build/Dockerfile .
 
-docker run \
-    -v /"$(pwd)"/docker/build/cache:/azerothcore/build \
+docker run --rm \
+    -v /"$(pwd)"/docker/build/cache:/azerothcore/var/build \
```

Comment:
not sure about this, have you synched it when it's referenced from other places?